osprey_flask_app/run_rvic.py

```diff
@@ -26,7 +25,7 @@ def run_full_rvic(arg_dict, url=get_target_url("osprey")):
             convolve_config_file=arg_dict["convolve_config_file"],
             convolve_config_dict=arg_dict["convolve_config_dict"],
         )
-    except Exception as e:
+    except Exception:
```

Comment:
Since you're not actually *handling* the exception here (just re-raising it), you can actually omit the whole try/except clause and the execution will be exactly the same.